sdk/resources/arm-features/package.json

```diff
@@ -26,7 +26,7 @@
     "rollup-plugin-sourcemaps": "^0.4.2",
     "uglify-js": "^3.4.9"
   },
-  "homepage": "https://github.com/azure/azure-sdk-for-js/tree/master/packages/@azure/arm-features",
+  "homepage": "https://github.com/azure/azure-sdk-for-js/tree/master/sdk/resources/arm-features",
```

Comment:
This didn't get updated.

Reply:
Nevermind it looks like I was looking at an older commit. The latest commit looks correct.

Reply:
Thanks!